EventsExpress.Core/Services/EventService.cs

```diff
@@ -230,7 +228,7 @@ public async Task<Guid> Edit(EventDto e)
                 .Include(e => e.EventSchedule)
                 .FirstOrDefault(x => x.Id == e.Id);
 
-            if (e.OnlineMeeting != null || e.Point != null)
+            if (e.Location != null)
             {
                 var locationDTO = Mapper.Map<EventDto, LocationDto>(e);
```

Comment:
do we still need this mapping? I guess `locationDTO` is stored as `e.location` already

Reply:
no, we don't need it any more, fixed